src/styles/blocks/header.scss

```
@@ -7,14 +7,15 @@
   height: 60px;
   padding: 0 50px;
 
-  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  box-shadow: 0 2px 4px $color-dark;
 
   &__logo {
     height: 40px;
   }
 }
 
 .nav {
+  transition: 300ms;
```

create a variable for transitionDuration
also, if you use transition, put the value & time transition to this style
Bc if you just put transition: 300ms  - it's like  === transition: all 0.3s
It's a too hard operation
```suggestion
  transition: color $transitionDuration;
```